Assets/MRTK/SDK/Experimental/PulseShader/Scripts/SurfacePulse.cs

```diff
@@ -1,37 +1,126 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
 using Microsoft.MixedReality.Toolkit.Utilities;
 using System.Collections;
-using System.Collections.Generic;
 using UnityEditor;
 using UnityEngine;
 
 namespace Microsoft.MixedReality.Toolkit.Experimental.SurfacePulse
 {
+    /// <summary>
+    /// Script for generating pulse shader effect on the surface.
+    /// </summary>
     [AddComponentMenu("Scripts/MRTK/SDK/SurfacePulse")]
     public class SurfacePulse : MonoBehaviour
     {
+        [SerializeField]
         [Tooltip("Shader parameter name to drive the pulse radius")]
-        public string ParamName = "_Pulse_";
```

Comment:
We might want to add `FormerlySerializedAs` tags for all these field renames.

Reply:
Added.